packages/babel-helpers/src/helpers.js

```diff
@@ -588,28 +588,46 @@ helpers.interopRequireDefault = helper("7.0.0-beta.0")`
 `;
 
 helpers.interopRequireWildcard = helper("7.0.0-beta.0")`
+  function _getRequireWildcardCache(obj) {
+    if (!global || typeof WeakMap !== 'function') {
```

Comment:
```suggestion
    if (typeof global === 'undefined' || typeof WeakMap !== 'function') {
```
otherwise you’ll still trigger bundler’s polyfills